pkg/kubelet/kubelet_test.go

```diff
@@ -193,6 +193,7 @@ func newTestKubeletWithImageList(
 	if err := os.MkdirAll(kubelet.rootDirectory, 0750); err != nil {
 		t.Fatalf("can't mkdir(%q): %v", kubelet.rootDirectory, err)
 	}
+	kubelet.podLogsDirectory = kubelet.rootDirectory
```

Comment:
this isn't correct, is it? I thought the logs dir was distinct from the root directory

Reply:
Well, the only place we test this is `TestKubeletDirs`, to make sure it returns the same value as assigned here, so creating a dedicated directory felt a bit excessive.

I can make it a subdirectory, like `kubelet.podLogsDirectory = filepath.Join(kubelet.rootDirectory, "pod-logs")`, is this makes more sense.

Reply:
@liggitt I've updated tests to assign separate directory for pod logs directory (also `t.TempDir` needs no `Mkdir` nor `Remove`), PTAL.